src/App.tsx

```diff
@@ -25,7 +27,7 @@ export default function App() {
         <Button
           secondary
           onClick={reloadExtension}
-          className="absolute top-3 right-3 cursor-pointer z-30"
+          className="absolute top-3 right-3 cursor-pointer z-40"
```

Comment:
change because of future overlay, just leave this change as it is, so I don't need to think in future again